app/code/Magento/InventoryConfigurableProduct/view/adminhtml/ui_component/product_form.xml

```diff
@@ -57,5 +57,16 @@
                 <externalFilterMode>true</externalFilterMode>
             </settings>
         </insertListing>
+        <component name="sources_visibility_checker" component="Magento_InventoryConfigurableProduct/js/components/sources-visibility-checker">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="formElement" xsi:type="string">container</item>
```

Comment:
Why do we need to have `formElement`as part of the configuration for the component?

Reply:
Removed.